CMakeLists.txt

```diff
@@ -283,6 +283,7 @@ endif ()
 include_directories (
     src/thirdparty/bcd
     src/thirdparty/lz4
+    src/thirdparty/murmurhash3
```

Comment:
We already have murmurhash in foundation/utility, wrapped in a class. It looks like the same version.
Use that one, modifying it if needed, instead of adding another copy of the code.

Reply:
Their specification requires "32 bit variant of MurmurHash3 (Also known as MurmurHash3_x86_32)"
The foundation version is a x64_128 bit version. I tried to use that hash truncating it to 32 bits but it didn't work. Is there any way I can get the same 32 bit value from 128bits?

Reply:
Maybe I should add x86_32 version to an existing `MurmurHash` class?

Reply:
I see. I think that we can keep it like it is, as an implementation detail of cryptomatte.

Reply:
Maybe I should remove all other implementations from external code and leave only x86_32bit?